config/web.php

```diff
@@ -38,19 +39,45 @@
             ],
         ],
         'db' => require(__DIR__ . '/db.php'),
-        /*
         'urlManager' => [
-            'enablePrettyUrl' => true,
             'showScriptName' => false,
+            'enablePrettyUrl' => true,
             'rules' => [
-            ],
+                '' => 'site/index',
+                '<action>'=>'site/<action>',
+                [
+                    'pattern' => '',
+                    'route' => 'site/index',
+                    'suffix' => ''
+                ],
+                [
+                    'pattern' => '<controller>/<action>/<id>',
+                    'route' => '<controller>/<action>',
+                    'suffix' => ''
+                ],
+                [
+                    'pattern' => '<controller>/<action>',
+                    'route' => '<controller>/<action>',
+                    'suffix' => '.html'
+                ],
+                [
+                    'pattern' => '<module>/<controller>/<action>/<id>',
+                    'route' => '<module>/<controller>/<action>',
+                    'suffix' => ''
+                ],
+                [
+                    'pattern' => '<module>/<controller>/<action>',
+                    'route' => '<module>/<controller>/<action>',
+                    'suffix' => '.html'
+                ]
+            ]
         ],
-        */
     ],
     'params' => $params,
 ];
 
-if (YII_ENV_DEV) {
+if (YII_ENV_DEV)
```

Comment:
According to the PSR-1 http://www.php-fig.org/psr/psr-2/#5-1-if-elseif-else 
it was correct to place open curly brace for if block on the first line with if statement
Why did you changed it?